### PR TITLE
test: fix coverage

### PR DESCRIPTION
**Pull request by viceice**
this pr fixes some coverage errors

blocked by #3795

### CODE REVIEW

**Comment by ViceIce** [comment]
@rarkins maybe we should fix some with tests instead of ignore?

**Comment by rarkins** [comment]
@ViceIce yeah, I'd definitely prefer enhancing tests rather than so many ignores

**Comment by ViceIce** [comment]
I'll try to add some tests.

File: lib/manager/bundler/update.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function updateDependency(currentFileContent, upgrade) {
       currentFileContent.split('"').length >
       currentFileContent.split("'").length
         ? '"'
-        : "'";
+        : /* istanbul ignore next */ "'";
```
Comment (ViceIce) [inline lib/manager/bundler/update.js]: @rarkins do we need to test this?

Comment (rarkins) [reply lib/manager/bundler/update.js]: Probably, yes. I guess it means we need a fixture with single quotes!

Comment (ViceIce) [reply lib/manager/bundler/update.js]: ok

File: lib/workers/pr/changelog/source-github.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
 async function getTags(endpoint, versionScheme, repository) {
   let url = endpoint
     ? endpoint.replace(/\/?$/, '/')
-    : 'https://api.github.com/';
+    : /* istanbul ignore next: not possible to test, maybe never possible? */ 'https://api.github.com/';
```
Comment (ViceIce) [inline lib/workers/pr/changelog/source-github.js]: @rarkins I think endpint can never be null or undefined here. Could not create a test for this.

**Comment by ViceIce** [comment]
@rarkins Ready to review 😄

**Pull request merged**